packages/nextjs/pages/index.tsx

```diff
@@ -247,9 +247,9 @@ const Home: NextPage<{
               </div>
             </div>
           </div>
-          <div className="flex flex-col items-center">
-            <div className="max-w-md lg:max-w-none">
-              <Image src="/assets/support-high-impact-devs.png" alt="developers list" width={900} height={900} />
+          <div className="flex flex-col items-center lg:pl-16 xl:pl-12 2xl:pl-24 lg:-mr-12 xl:-mr-16 2xl:-mr-24">
+            <div className="max-w-md lg:max-w-md xl:max-w-xl 2xl:max-w-2xl">
+              <Image src="/assets/streams.png" alt="developers list" width={700} height={700} />
```

Comment:
Nitpick: you can update the `alt` tag to reflect the new image. We could either describe it literally (e.g. "Hourglass overlooking landscape") or describe the symbolism.